Zilon.Core/Zilon.Core/Tactics/StaticObject.cs

```diff
@@ -36,13 +37,20 @@ public StaticObject(IGraphNode node, PropContainerPurpose purpose, int id)
 
         /// <inheritdoc/>
         public PropContainerPurpose Purpose { get; }
+        public PhysicalSize PhysicalSize { get => PhysicalSize.Size1; }
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Add a way to break out of this property accessor's recursion.](https://app.codacy.com/manual/kreghek/Zilon_Roguelike/pullRequest?prid=5403398)